src/publish.proj

```diff
@@ -24,4 +28,21 @@
     <!-- skip uploadToAzure when there are no nupkgs -->
     <CallTarget Targets="UploadToAzure" Condition="'@(_PackagesToPublish)' != ''" />
   </Target>  
+  
+  <Target Name="FinalPublish" >
```

Comment:
Didn't we decide to call this PublishToAzureBlob or something along those lines?